packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -2,7 +2,9 @@
 	position: relative;
 	display: flex;
 	flex-flow: column;
-	overflow: hidden;
+	&.is-isolated {
+		overflow: hidden;
```

Comment:
Does it hurt to always apply this rule?

Reply:
Yes, unfortunately `overflow: hidden` in the non-iframed editor prevents the styling for sticky positioned blocks from working.

Reply:
Oops, sorry, I meant to never apply this rule. There's no comment about why it's needed

Reply:
> Oops, sorry, I meant to never apply this rule. There's no comment about why it's needed

Ah, gotcha. And thanks for adding the explanatory comment @stokesman!

@ellatrix there's a screengrab on the earlier PR that added `overflow: hidden` over in https://github.com/WordPress/gutenberg/pull/49978 — basically, the interaction between the iframed editor and the block toolbar popover results in additional scrollbars when the toolbar goes off screen in long posts if we don't have the rule in place.